samples/dnn/text_detection_and_recognition.cpp

```diff
@@ -0,0 +1,241 @@
+//  This example can not only detect text through the camera, but also perform real-time text recognition. Finally, display the results in the bounding box.
```

Comment:
What's the difference with https://github.com/opencv/opencv/blob/master/samples/dnn/text_detection.cpp?

Reply:
So far, the difference is the text recognition model. I use VGG, and text_detection.cpp uses CRNN.
And next, I will provide more text recognition model support (like ResNet+BLSTM, DensNet, and etc.) and better text detection methods.
Finally, I will analyze the performance of these models on OpenCV in terms of accuracy and speed.

Reply:
But text detection model is the same which leads to code duplication. Can you please add new text recognition model into this sample?

Reply:
Ok, I have added a link into text_detection.cpp.